### DISCUSSION TITLE
Requires plugin 'com.redhat.devtools.lsp4ij' to be installed

### DISCUSSION THREAD

**Discussion by seachicken** [redhat-developer/lsp4ij#286] [Q&A] [answered]
Thanks for the useful tools. I'm currently developing a plugin with lsp4ij.
https://github.com/seachicken/intellij-inga/blob/be87a5af0e91528466c8b1ffeb85f6b07f482876/build.gradle.kts#L22
 I ran `./gradlew publishPlugin` and installed the plugin from disk then I'm getting an error:

<img width="998" alt="スクリーンショット 2024-05-09 20 20 45" src="https://github.com/redhat-developer/lsp4ij/assets/5178598/3a474a30-9143-4fd1-b116-237aa8dd7d69">

If `https://plugins.jetbrains.com/plugins/nightly/23257` repository is added, then working. but this setting is not normal use for users. How do you solve this with [intellij-quarkus](https://github.com/redhat-developer/intellij-quarkus) etc.?

**Comment by angelozerr** [comment, marked as answer]
It is a great news for LSP4IJ which is not released yet :)

Could you please create a PR to add your Intellij plugin at https://github.com/redhat-developer/lsp4ij?tab=readme-ov-file#who-is-using-lsp4ij please

I think your problem comes from that LSP4IJ is not released but we should do the release soon.

**Comment by seachicken** [comment]
Thank you, that's good news. I'll try to create a PR later :)
